### PR TITLE
[DOC] Consumer lag image

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

Returned to the Kafka Exporter section describing consumer lag.
Thought an image might be useful to show the delta.

The image is a draft.
If approved, I'll update with one consistent with look of other images in the docs.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by PaulRMellor** [comment]
> Out of curiosity, why are we calling it "delta" as well, instead of just "lag" (on the picture too)?
> I think that "lag" is the common term used in the Kafka ecosystem for this.

We can change the label to **_lag_** on the image. But we're talking specifically about the _gap_ not lag generally.

**Comment by ppatierno** [comment]
I see your point but Kafka users don't know about "delta" ... even "gap", they all talk about lag.

**Comment by PaulRMellor** [comment]
> I see your point but Kafka users don't know about "delta" ... even "gap", they all talk about lag.

Okay. I'll change the label to _lag_ :+1:

File: documentation/modules/metrics/kafka-exporter/con_kafka-exporter-lag.adoc [hunk 1]
```diff
@@ -8,14 +8,15 @@
...
 
-This difference is sometimes referred to as the _delta_ between the producer offset and consumer offset, the read and write positions in the Kafka broker topic partitions.
+The lag reflects the position of the consumer offset in relation to the end of the partition log.
 
-.Consumer lag (delta) between the producer and consumer offset
+.Consumer lag between the producer and consumer offset
 
 image:consumer-lag.png[Consumer lag]
 
+This difference is sometimes referred to as the _delta_ between the producer offset and consumer offset, the read and write positions in the Kafka broker topic partitions.
```
Comment (laidan6000) [inline documentation/modules/metrics/kafka-exporter/con_kafka-exporter-lag.adoc:18]:
```suggestion
This difference is sometimes referred to as the _delta_ between the producer offset and consumer offset: the read and write positions in the Kafka broker topic partitions.
```

**Comment by scholzj** [comment]
@ppatierno @PaulRMellor Can we get this PR over the line? It looks like Paolo's approval and one small comment from Daniel are only things missing.

**Comment by ppatierno** [comment]
@PaulRMellor sorry if I am so picky on this. You say something like "....indicates the delay between the last message....". The delay word let me think it's a "time" difference not a "position/offset" difference which is what the "lag" actually is.

**Comment by scholzj** [comment]
@ppatierno My two cents ... the lag **is** possition/offset difference ... but it **indicates** the delay (the higher the lag, the higher the delay). (just saying that for me this is not wrong - if we leave it out I don't have problem with it of course)

**Comment by PaulRMellor** [comment]
> @PaulRMellor sorry if I am so picky on this. You say something like "....indicates the delay between the last message....". The delay word let me think it's a "time" difference not a "position/offset" difference which is what the "lag" actually is.

@ppatierno @scholzj -- _delay_ doesn't just mean time difference, it can mean impede or hinder progress. I think we're okay here?  :man_shrugging:

**Pull request merged**